optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc

```diff
@@ -132,6 +132,8 @@ https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/html/#run
 You can learn more about configuring a Spring Boot application on the
 https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config[Spring Boot Externalized Configuration] page.
 
+Source files are automatically formatted during a maven build, so you do not need to configure a formatter for your IDE.
```

Comment:
But what if I want to format the code in the IDE?

I find the IDE setup instructions useful and have used it several times. We should keep it somewhere. What's the problem with the original location (`ide-configuration/ide-configuration.adoc`)?

Reply:
We can keep the IDE setup instructions, but in my experience, the maven formatter sometimes still make changes to code formatted by an IDE (either due to the formatting instructions not being one-to-one or otherwise). As such, code should be `mvn compile` before committing. Any IDE configuration for formatting should be described as optional. "Source files are automatically formatted during a maven build" should be mention somewhere (since it is unexpected).

Reply:
Yes, the automatic formatting during Maven build should be mentioned. I agree.

Another important message I'd like to pass to the reader is: "You don't have to run Maven in order to format the code. You can still use your IDE but instead of importing an IDE-native style config, you should use the Eclipse Formatter plugin for your IDE."

Note that this applies in the context of Java modules. In TypeScript modules, the approach is the same. The source code is formatted automatically during npm build with ESLint. But developers are still encouraged to format their code frequently in the IDE. In this case using the ESLint plugin.

Now, the question is where to put this information. I think the full information should be included in the development chapter in the documentation. We may also point there from CONTRIBUTING.md on GitHub.

Reply:
Quarkus contributing guide (example): https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#ide-config-and-code-style.

Reply:
I basically copied/pasted the Quarkus guide with references change (Quarkus => The backend, updated links to ide-config, converted to adoc)